### PR TITLE
10 add type to script tag - improvement but not fixed

### PR DESCRIPTION
**Pull request by MayaGans**
Attempting #10 [Not sure if making it better or worse] 

There's no longer the error about the `<`, but  there's a new error 

```Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at file:///Users/mayagans/Documents/JavaScript/js-vs-ds/scratch/app.js. (Reason: CORS request not http).```

 and I don't get the expected/desired error for the race issue.

### CODE REVIEW

**Comment by MayaGans** [comment]
I'm getting the same error with Hello Bottom (and don't see the React component)

File: dynamic.tex [hunk 1]
```diff
@@ -299,6 +299,22 @@ \section{Developing with Parcel}\label{s:dynamic-parcel}
...
+but we have not yet used the node package manager, \texttt{npm}. 
+Before we install Parcel, 
+we can turn the directory which we've been creating our html and js files into a project
+by typing 
+
+\begin{minted}{shell}
+$ npm init -y
+\end{minted}
+
+\end{aside}
```
Comment (gvwilson) [inline dynamic.tex]: It looks like this includes the changes made in #98 as well?

File: dynamic.tex [hunk 2]
```diff
@@ -382,7 +398,7 @@ \section{Multiple Files}\label{s:dynamic-multiple}
   </head>
   <body>
     <h1>Hello Separate</h1>
-    <div id="app"></div>
+    <div id="app" type="text/babel"></div>
```
Comment (gvwilson) [inline dynamic.tex]: :+1:

Comment (gvwilson) [inline dynamic.tex]: Why does this `div` need to have `type="text/babel"`? There's no JavaScript inside this `div` that needs to be translated.

File: src/dynamic/hello-separate/index.html [hunk 3]
```diff
@@ -6,7 +6,7 @@
     <script src="https://fb.me/react-15.0.1.js"></script>
     <script src="https://fb.me/react-dom-15.0.1.js"></script>
     <script src="https://unpkg.com/babel-standalone@6/babel.js"></script>
-    <script src="app.js"></script>
+    <script src="app.js" type="text/babel"></script>
```
Comment (gvwilson) [inline src/dynamic/hello-separate/index.html]: *This* needs `type="text/babel"` because there *is* JS to translate.

**Comment by MayaGans** [comment]
Oops, got a little sloppy with my copy paste since the div and js file have the same name - good catch, thanks. I also changed the type within Hello Bottom both in the src folder as well as inline in dynamic.tex

**Comment by MayaGans** [comment]
To keep consistant I also added the type to hello-parcel with src and in the text.

**Comment by gvwilson** [comment]
Hiya - can you please take another crack at this some time this week or next?

**Pull request merged**